error-prone-contrib/pom.xml

```diff
@@ -216,6 +216,12 @@
             <artifactId>testng</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>${version.lombok}</version>
```

Comment:
Small tip; if we only have one version declaration as we have here, we don't need to extract it to a `<version.lombok>`.